spec/tools/environment_builders/openstack/services/storage/data.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../base_data'
+
+module Openstack
+  module Services
+    module Storage
+      class Data < ::Openstack::Services::BaseData
+        def directory_name_1
+          "dir_1"
```

Comment:
nah, as you say, just tools. :-) I might be changing this when expanding number of tested resoruces. I might change them to constants, since that is what is should be. :-)